src/constr.f90

```diff
@@ -613,7 +613,11 @@ Subroutine dphidrPBC(mode,nat,xyz,i,j,k,l,vTrR,vTrB,vTrC,phi,&
       dphidrj=0
       dphidrk=0
       dphidrl=0
-      onenner=1.0d0/(nan*nbn)
+      if (abs(nan*nbn).gt.eps) then
```

Comment:
It might be worth clarifying what the purpose of this check is. It seems like we would only reach this condition if for some reason the magnitude of one or both of these surface normals was very small. The earlier check already determines if the normals are nearly parallel.

Is there a specific example you have found that the code change here fixes?

Reply:
>  The earlier check already determines if the normals are nearly parallel.

You can be here if `nan` or `nbn` equal zero and then you divide by zero :) 

> Is there a specific example you have found that the code change here fixes?

Yep, you can detect this with input provided in #500.

Reply:
I'm surprised the example from https://github.com/grimme-lab/xtb/issues/500 goes through this function. I assumed from the name it only applied for PBC structures, but the example from the issue is molecular.

Reply:
¯\_(ツ)_/¯